OG_Format.adoc

```diff
@@ -665,21 +655,23 @@ Where <sensor_type> is a controlled vocabulary: see vocabulary guidelines
 
 string SENSOR_<sensor_type>_<sensor_serial_number>
 
-SENSOR_<sensor_type>_<sensor_serial_number>:sensor_type_vocabulary = “” - mandatory
+attributes:
 
-SENSOR_<sensor_type>_<sensor_serial_number>:long_name: “” - mandatory 
+sensor_type_vocabulary: “https://vocab.nerc.ac.uk/collection/L05/current/” - mandatory
```

Comment:
see above

Reply:
I've created a issue with regard to this topic. I'll close the conversation to facilitate merging.